test/integration/transaction_test.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import pyexasol
+
+
+# For the fetch_dict tests we need to configure the connection accordingly (fetch_dict=True)
```

Comment:
would it makes sense to add this as docstring

Reply:
No but I see the comment is wrong ... thanks :+1: